corehq/apps/hqadmin/reports.py

```diff
@@ -495,3 +502,92 @@ def headers(self):
     @property
     def rows(self):
         return self.table_data.rows
+
+
+class StaleCasesTable:
+    STOP_POINT_DAYS_AGO = 365 * 20
+    AGG_DATE_RANGE = 150
+    STALE_DATE_THRESHOLD_DAYS = 365
+
+    BACKOFF_AMOUNT_DAYS = 30
+    MAX_BACKOFF_COUNT = 2
+
+    def __init__(self):
+        self._rows = None
+        self.stop_date = datetime.now() - timedelta(days=self.STOP_POINT_DAYS_AGO)
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(gettext_lazy("Domain")),
+            DataTablesColumn(gettext_lazy("Case count"))
+        )
+
+    @property
+    def rows(self):
+        if self._rows is None:
+            self._rows = []
+            case_count_by_domain = self._aggregate_case_count_data()
+            for domain, case_count in case_count_by_domain.items():
+                self._rows.append([domain, case_count])
+        return self._rows
+
+    def _aggregate_case_count_data(self):
+        end_date = datetime.now() - timedelta(days=self.STALE_DATE_THRESHOLD_DAYS)
+        agg_res = {}
+        curr_backoff_count = 0
+        curr_agg_date_range = self.AGG_DATE_RANGE
+        domains = self._get_domains()
+        while (True):
+            start_date = end_date - timedelta(days=self.AGG_DATE_RANGE)
```

Comment:
Should this be `curr_agg_date_range` instead of `self.AGG_DATE_RANGE` ?

Reply:
Addressed in [b228b40](https://github.com/dimagi/commcare-hq/pull/35446/commits/b228b408ee63cb87d5786417b375e517c54afcd6)